stable/metrics-server/README.md

```diff
@@ -13,3 +13,4 @@ Parameter | Description | Default
 `image.repository` | Image repository | `gcr.io/google_containers/metrics-server-amd64`
 `image.tag` | Image tag | `v0.2.1`
 `image.pullPolicy` | Image pull policy | `IfNotPresent`
+`kubeletReadOnlyPort` | Set to `false` if the kubelet read-only port is disabled (it is disabled by default in K8S 1.11+) | `true` |
```

Comment:
This isn't really how I want this solved. The default port metrics-server will use will be changed to the secure one soon. See https://github.com/kubernetes-incubator/metrics-server/issues/77#issuecomment-404950375